Buildings/Resources/Scripts/EnergyPlus/energyplus_csv_to_mos.py

```diff
@@ -1,13 +1,14 @@
 #!/usr/bin/env python3
 
-def energyplus_csv_to_mos(output_list, dat_file_name, step_size, final_time):
+def energyplus_csv_to_mos(output_list, dat_file_name, step_size, final_time, start_time:int = 0):
```

Comment:
Is there a reason you use typing for one argument (`start_time:int`) but don't use typing anywhere else in the whole file? Please use consistent style in your code.